releasenotes/notes/delay-fake-backends-3f68c074e85d531f.yaml

```
@@ -0,0 +1,12 @@
+---
+fixes:
+  - |
+    The :class:`~.BackendV2` based fake backends in
+    the :mod:`qiskit.providers.fake_provider` module, such as
+    ``FakeMontrealV2``, previously were missing the :class:`~.Delay` operation
+    support from their :attr:`~.BackendV2.target` attribute. This prevented
+    compiling some :class:`~.QuantumCircuit` objects that contained
+    :class:`~.Delay` instructions to these backends. This has been corrected
+    by adding the :class:`~.Delay` to the :class:`~.Target` object for each
+    fake backend to ensure that the compiler knows that :class:`~.Delay` is
+    a valid operation on the fake backends.
```

Sigh: https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=36199&view=logs&j=abfb3e93-d1ff-5b1b-cb4c-74d6dcf536f9&t=60edc4b3-4259-59c8-ae13-dc31febe2d3d&l=7522

```suggestion
    The :class:`~.BackendV2` based fake backends in
    the :mod:`qiskit.providers.fake_provider` module, such as
    ``FakeMontrealV2``, previously were missing the :class:`~qiskit.circuit.Delay` operation
    support from their :attr:`~.BackendV2.target` attribute. This prevented
    compiling some :class:`~.QuantumCircuit` objects that contained
    :class:`~qiskit.circuit.Delay` instructions to these backends. This has been corrected
    by adding the :class:`~qiskit.circuit.Delay` to the :class:`~.Target` object for each
    fake backend to ensure that the compiler knows that :class:`~qiskit.circuit.Delay` is
    a valid operation on the fake backends.
```